src/main/java/module-info.java

```diff
@@ -0,0 +1,6 @@
+module io.avaje.prism {
+
+  exports io.avaje.prism;
+  requires java.compiler;
+
```

Comment:
I'm thinking we need to add the `provides javax.annotation.processing.Processor with ... `

Reply:
I'll merge this PR and add that in I reckon ...

Reply:
ah yeah forgot about that